frontend/src/components/Teams.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { Image, CloudinaryContext } from "cloudinary-react";
+import { backend } from "../conf.js";
+import "./style/Teams.scss";
+
+function Team() {
+  const [teams, getTeams] = useState([]);
+  useEffect(() => {
+    axios
+      .get(`${backend}/api/teams/`)
+      .then(({ data }) => {
+        getTeams(data);
+      })
+      .catch(err => {});
+  }, []);
+  return (
+    <div className="teamPage">
+      <h1>Teams</h1>
+      {teams
+        ? teams.map(team => {
```

Comment:
This ternary is useless : if `teams` is empty, the `map` will be executed 0 time, resulting in a `null` value